pom.xml

```diff
@@ -97,10 +107,31 @@
             <dependency>
               <groupId>org.apache.maven.surefire</groupId>
               <artifactId>surefire-junit47</artifactId>
-              <version>3.0.0-M5</version>
+              <version>3.0.0-M7</version>
             </dependency>
-          </dependencies>
+          </dependencies>     
```

Comment:
Should probably revert these changes?

Reply:
updated the version as the older version of 3.0.0-M5 is failing